backend/tsconfig.json

```diff
@@ -17,7 +17,10 @@
     "skipLibCheck": true,
     "typeRoots": [
       "./src/types",
-      "./node_modules/@types"
+      "./node_modules/@types",
+      "./node_modules/@google-cloud/secret-manager/build/src",
+      "./node_modules/@google-cloud/service-usage/build/src",
+      "./node_modules/@google-cloud/resource-manager/build/src"
```

Comment:
I don't think you need to add this here. Can you try removing and see what happens?

Reply:
I added it here as there are some interfaces and namespaces from gcp-libraries, I wanted to use. 
For example:- In this file `backend/src/integrations/sync.ts`
Before, adding these typeRoots, I was getting type-errors.